dbms/src/Functions/FunctionsGrouping.h

```diff
@@ -191,6 +230,8 @@ class FunctionBuilderGrouping : public IFunctionBuilder
 
     String getName() const override { return name; }
     bool useDefaultImplementationForNulls() const override { return true; }
+    // at frontend, grouping function can receive maximum number of parameters as 64.
+    // at backend, grouping function has been rewritten as receive only gid with meta.
     size_t getNumberOfArguments() const override { return 1; }
```

Comment:
As the number of input parameter is variable, the return number should be changed

Reply:
It's ok here, the pb function that tiflash received only has one arg (gid) with meta